kairon/api/app/routers/user.py

```
@@ -26,12 +26,14 @@ async def get_users_details(current_user: User = Depends(Authentication.get_curr
     return {"data": {"user": user_details}}
 
 
-@router.post("/details", response_model=Response)
-async def update_users_details(current_user: User = Depends(Authentication.get_current_user)):
+@router.post("/details/{status}", response_model=Response)
+async def update_user_details(
+        status: str = Path(description="user onboarding status", examples=["Completed", "Skipped"]),
+        current_user: User = Depends(Authentication.get_current_user)):
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)